src/components/sidemenu/SideMenuPanel.js

```diff
@@ -82,6 +88,27 @@ const getMenuItems = ({ API, hideSidemenu, showDialog, navigation, store, theme
           hideSidemenu()
         },
       },
+      {
+        icon: 'add',
+        name: 'Add app Icon',
+        hidden: !installPrompt && !isMobileSafari,
+        size: 18,
```

Comment:
why does it need custom size? icons should be the same size by default. verify with Liav.

Reply:
I used custom size because the round icons are visually larger and resizing was a Liav initiative. @LiavGut, can you give me the icons of the same height, so as not to customize the size of the icons

Reply:
ok liav will make new ones